mx.py

```diff
@@ -4772,9 +4777,8 @@ def expand(value, context):
             'developer',
             'url',
             'licenses',
-            'licences',
             'defaultLicense',
-            'defaultLicence',
```

Comment:
Why did you remove these two keys? What effect does removing them have?
(I'm not familiar with that part of the code... @gilles-duboscq, do you know what this does?)